Dockerfile.ci

```diff
@@ -0,0 +1,13 @@
+FROM ocaml/opam:alpine-3.4_ocaml-4.03.0
+RUN sudo apk add ncurses-dev tzdata
```

Comment:
Confirmed: all you need now to get libev support is: `opam depext -i lwt conf-libev`